Daemon Library.cat

```diff
@@ -21,7 +21,7 @@ A unit which includes models with this special rule without a specific Ætheric
           </constraints>
           <rules>
             <rule id="4438-874e-901d-f86" name="Encroaching Ruin" publicationId="8775-88f5-cfdd-24f6" page="3" hidden="false">
-              <description>When a unit entirely composed of models with this special rule and the Daemon Unit Type fails a Morale check, it only suffers 1 automatic Wound with no Saves of any kind allowed instead of D3 Wounds as specified in the Daemon Unit Type. In addition, models with this special rule gain the Move Through Cover special rule.</description>
+              <description>When a unit entirely composed of models with this special rule and the Daemon Unit Type fails a Morale check, it only suffers 1 automatic Wound with no Saves of any kind allowed instead of D3 Wounds as specified in the Daemon Unit Type. In addition, models with this special rule gain the Move Through Cover special rule.</description>
```

Comment:
I thought we'd move nonbreaking spaces to their own PR.

If so, that should be there

Reply:
NBSP was merged already. This change appears to be " " to "nbsp". These are ones I recopied from the book to ensure I had the most recent wording, and that's why they have nbsp in them. Fixing.